.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!--Thanks for your contribution. See [CONTRIBUTING](CONTRIBUTING.md)
+    for Pulumi's contribution guidelines.
+
+    Help us merge your changes more quickly by adding more details such
```

Comment:
Is this something external contributors can set?

Reply:
Hmm good point, I'm not sure. I copied this from https://github.com/pulumi/docs/blob/master/.github/ISSUE_TEMPLATE.md

Reply:
They should be able to, but I haven't personally tested with a different account.